toolkit/tools/imager/imager.go

```diff
@@ -473,12 +474,23 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap m
 		defer installutils.DestroyInstallRoot(installRoot, installMap, mountPointToOverlayMap)
 	}
 
+	// Install any tools required for the setup root to function
+	setupChrootPackages := installutils.GetRequiredPackagesForInstall()
+	for _, setupChrootPackage := range setupChrootPackages {
+		pkg := setupChrootPackage.Name
+		_, err = installutils.TdnfInstall(pkg, rootDir)
+		if err != nil {
+			err = fmt.Errorf("failed to install required setup chroot package '%s': %w", pkg, err)
+			return
+		}
+	}
```

Comment:
What do you think about merging this with the step below? What I have in mind: start with `setupChrootPackages`, append the verify packages to it, if needed, and then install everything in one loop.